templates/prometheus.yml.j2

```diff
@@ -22,16 +22,20 @@ scrape_configs:
     static_configs:
       - targets: ['speedtest:9798']
 
+{% if not shelly_plug_enable %}      
```

Comment:
Wouldn't this be `if shelly_plug_enable`? The `not` would mean it appears only if `shelly_plug_enable: false`...

Reply:
Absolutely, apologies. I pushed a version from before a test run. Caught the nots and a shelly_plug_enable name issue in the task.